### DISCUSSION TITLE
Navigate without context

### DISCUSSION THREAD

**Discussion by udaymrx** [csells/go_router#276] [Q&A] [answered]
Add ability to navigate without the need of context, 
It will be really helpful at places like controller, where we don't have context and need to do some kind of navigation (to seperate the business logic from the UI completely), but we can't.

**Comment by csells** [comment, marked as answer]
You don't need context to call `GoRouter.go()` or `GoRouter.push`; you can make your `GoRouter` instance global and call it from wherever you need.
